api/v1alpha1/ramenconfig_types.go

```diff
@@ -132,6 +132,12 @@ type RamenConfig struct {
 		// cluster service version name
 		ClusterServiceVersionName string `json:"clusterServiceVersionName,omitempty"`
 	} `json:"drClusterOperator,omitempty"`
+
+	// VolSync configuration
+	VolSync struct {
+		// Disabled is used to disable VolSync usage in Ramen. Defaults to false.
+		Disabled bool `json:"disabled,omitempty"`
```

Comment:
After thinking more about it, I think it is better to have it as a string. That would allow us to have more than one enabled options later, like volsyncPreferred, volsyncAsFallback etc.

I am okay with this change being done in a follow up PR, but good to create an issue now.

Reply:
A new issue has been created to track this [here](https://github.com/RamenDR/ramen/issues/457)